src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs

```diff
@@ -1,14 +1,12 @@
-﻿using System.Collections.Generic;
-using Sfa.Das.Sas.Web.Extensions;
+﻿using System.Text;
 
 namespace Sfa.Das.Sas.Web.Services.MappingActions.Helpers
 {
+    using System.Collections.Generic;
     using System.Linq;
-
+    using ApplicationServices.Models;
     using Microsoft.Ajax.Utilities;
 
-    using Sfa.Das.Sas.ApplicationServices.Models;
-
     public static class ProviderMappingHelper
     {
         public static string GetPercentageText(double? level)
```

Comment:
method can be simplified into a single return statement:
--
return level == null ? "no data available" : $"{level}%";
--